src/Interactive/Host/Interactive/Core/InteractiveHost.LazyRemoteService.cs

```diff
@@ -123,7 +123,12 @@ private async Task<InitializedRemoteService> TryStartAndInitializeProcessAsync(C
 
                     return new InitializedRemoteService(remoteService, result);
                 }
+#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods
+                // await ExecuteRemoteAsync above does not take cancellationToken
+                // - we don't currently support cancellation of the RPC call,
+                // but JsonRpc.InvokeAsync that we use still claims it may throw OperationCanceledException..
                 catch (Exception e) when (FatalError.ReportAndPropagateUnlessCanceled(e))
+#pragma warning restore CA2016
```

Comment:
💭 Since changing the call to `ReportAndPropagateUnlessCanceled` only impacts telemetry, it seems fine to pass the cancellation token to the filter and be aware through telemetry if we are often producing unexpected cancellations.

Reply:
@tmat it looks like you requested the current form. I believe the original form of this PR better met expectations.